format/FormatNumber.js

```diff
@@ -276,4 +276,12 @@ export const numberRenderer = createRenderer(fmtNumber),
     billionsRenderer = createRenderer(fmtBillions),
     quantityRenderer = createRenderer(fmtQuantity),
     priceRenderer = createRenderer(fmtPrice),
-    percentRenderer = createRenderer(fmtPercent);
\ No newline at end of file
+    percentRenderer = createRenderer(fmtPercent);
+
+export const fmtNumberEl = createElFormatter(fmtNumber),
+    fmtThousandsEl = createElFormatter(fmtThousands),
+    fmtMillionsEl = createElFormatter(fmtMillions),
+    fmtBillionsEl = createElFormatter(fmtBillions),
+    fmtQuantityEl = createElFormatter(fmtQuantity),
+    fmtPriceEl = createElFormatter(fmtPrice),
+    fmtPercentEl = createElFormatter(fmtPercent);
```

Comment:
Am trying to think if I have any other suggestions for the names here.  Was thinking just `fooEl` to match `fooRenderer` - that sounds OK with e.g. `dateTimeEl` but less so with `millionsEl`....

Also kinda tempted to just drop any qualifier and use `dateTime` or `millions` directly...  Given that you are always importing from a format package it doesn't seem entirely crazy...  

Another option would be `dateTimeFormatter` or `millionsFormatter`.  Seems more readable / natural in some ways, although long.

So I can see why you landed where you did.  The names are a bit awkward but also probably not that common.

Was that helpful? 😄

Reply:
Two things:


+ I think the 'fooRenderer' convention itself is targeting Ext and could be reworked.
+ How about 'quantityFmt' and 'quantityFmtEl'?  Like the idea of not splitting the modifier around the variable name?

Maybe a quick discussion of this is in order?